vlib/v/checker/tests/fn_var.out

```diff
@@ -1,3 +1,8 @@
+vlib/v/checker/tests/fn_var.vv:1:10: error: missing return at end of function `anon_27_7_9`
```

Comment:
I don't like the name `anon_27_7_9`. Can anyone suggest a better way to use the proper naming for `ast.AnonFn`'s name. I know we have to use `typ` field of it but how will I pass it through?

Reply:
just `anonymous function` ?

Reply:
`error: missing return at the end of an anonymous function`

Reply:
I thought of doing that but it will look inconsistent with other functions.

Reply:
`error: missing return at the end of an anonymous function` is fine

Reply:
Ok

Reply:
i think we have something that makes `fn (v int) bool` possible (the function signature).

Reply:
@danieldaeschle how will this be done in without `table.type`?